crates/egui_demo_lib/src/demo/widget_gallery.rs

```diff
@@ -209,7 +209,10 @@ impl WidgetGallery {
         let img_size = 16.0 * texture.size_vec2() / texture.size_vec2().y;
 
         ui.add(doc_link_label("Image", "Image"));
-        ui.raw_image((texture.id(), img_size));
+        ui.allocate_ui(img_size * 2.0 + egui::Vec2::new(8.0, 0.0), |ui| {
+            ui.image((texture.id(), img_size));
+            ui.image(egui::include_image!("../../assets/icon.png"));
+        });
```

Comment:
Remove the old `texture` and just use the new API

```suggestion
        let egui_icon = egui::include_image!("../../assets/icon.png");
        ui.add(egui::Image(egui_icon).max_size(egui::Vec2::splat(16.0));
```

Reply:
Also, if the icon is only being used here, we can downsize the .png quite a bit, e.g. to 32x32. It will improve its visual quality too since we don't do mip-mapping on the renderer yet.